docs/docs/data-sources/smtp.md

```diff
@@ -0,0 +1,12 @@
+
+# smtp
+
+ToolJet can connect to SMTP Server databases to read and write data. 
```

Comment:
ToolJet can connect to SMTP server to send emails.

Reply:
It's auto-generated content. I have added a new doc just a minute ago